src/main/java/com/intuit/graphql/orchestrator/federation/validators/FieldSetValidator.java

```diff
@@ -49,6 +50,33 @@ public void validate(XtextGraph sourceGraph, TypeDefinition typeDefinition, Stri
         }
     }
 
+    public void validate(XtextGraph sourceGraph, TypeExtensionDefinition typeExtensionDefinition, String fieldSet, String originatingDirective) {
+        Objects.requireNonNull(sourceGraph);
+        Objects.requireNonNull(typeExtensionDefinition);
+        if(StringUtils.isBlank(fieldSet)) {
+            throw new EmptyFieldsArgumentFederationDirective(typeExtensionDefinition.getName(), originatingDirective);
+        }
+
+        if(!fieldSet.startsWith("{")) {
```

Comment:
Was reviewing the FieldSet specs.. from this definition:
```
Grammatically, a field set is a [selection set](http://spec.graphql.org/draft/#sec-Selection-Sets) minus the braces.
```
Looks like we should fail if it starts with '{'

I'm thinking 
1.  We should have a grammar for FieldSet, or
2.  Let's just join straight { fieldset }then let the Parser.parser() accept or fail.